contracts/Forwarder.sol

```diff
@@ -93,6 +93,11 @@ contract Forwarder is ReentrancyGuard, ERC721TokenReceiver {
     return this.onERC721Received.selector;
   }
 
+  function callFromParent(address target, uint256 value, bytes calldata data) external nonReentrant onlyParent {
```

Comment:
are we calling this function using sendMultiSig from wallet?

Reply:
yes, that would be where we call this from